src/node/board.js

```diff
@@ -0,0 +1,79 @@
+// depois do code review eu apago os comentarios. é uma "boa pratica"?
+module.exports = {
+  createMatrix,
+  viewBoard,
+  // duvida: aqui eu preciso passar as funções "auxiliares" da função principal?
+};
+
+function createMatrix(size) {
+  const matrix = []; // matrix me parece um nome ruim mas é oq realmente é, a matrix do tabuleiro, nao pensei em nada melhor
+  const line = [];
+  for (let i = 0; i < size; i += 1) {
+    for (let j = 0; j < size; j += 1) {
```

Comment:
Não precisa dar esse passo agora, mas a gente quase não usa for loops. Para arrays a gente usa muito mais `.map`. Sugiro tentar dar uma lida depois.